Modules/Microsoft365DSC/Modules/M365DSCStubsUtility.psm1

```diff
@@ -38,21 +38,21 @@ function New-M365DSCStubFiles
         }
     }
 
-    $currentPath = Join-Path -Path $PSScriptRoot -ChildPath '..\' -Resolve
-    $manifest = Import-PowerShellDataFile "$currentPath/Microsoft365DSC.psd1"
-    $dependencies = $manifest.RequiredModules
     $Content = ''
     $folderPath = Join-Path $PSScriptRoot -ChildPath "../DSCResources"
     Write-Host $FolderPath
-    $workloads = @('ExchangeOnline', 'SecurityComplianceCenter', 'PnP', 'PowerPlatforms', 'MicrosoftTeams', 'MicrosoftGraph')
+    $workloads = @(
+        <#'ExchangeOnline', 'SecurityComplianceCenter', 'PnP', 'PowerPlatforms',#> @{Name = 'MicrosoftTeams'; ModuleName = "MicrosoftTeams"}<#, 'MicrosoftGraph'#>
```

Comment:
Is it correct that the other workloads are commented out?

Reply:
Done